modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestForm/steps/MdfList/components/columns.tsx

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
It should be good to use those columns inside a function or the component directly